connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorSourceConnectorTest.java

```diff
@@ -152,6 +152,44 @@ public void testConfigPropertyFiltering() {
             .anyMatch(x -> x.name().equals("min.insync.replicas")), "should not replicate excluded properties");
     }
 
+    @Test
+    public void testConfigPropertyFilteringExclude() {
```

Comment:
@mimaison, thanks for the comments. Makes perfect sense.
I'm adding integration test to address your suggestion.
Along with that, i'm still keeping the unitTest committed before, so it will test configuration filtering _logic_ without integration. The method I've added to unitTest follows the suit of the very previous test `testConfigPropertyFiltering`, which tests **default** exclude list this particular way.
I understand your concern of using connector's configuration instead of topic configuration. But since in unitTest we don't really create cluster's to retrieve topic's configuration, I find this way legit to test `MirrorSourceConnector.targetConfig`. Open for suggestions, though.